misc/loop/go.mod

```diff
@@ -7,43 +7,36 @@ toolchain go1.22.4
 require (
 	github.com/docker/docker v24.0.7+incompatible
 	github.com/docker/go-connections v0.4.0
-	github.com/gnolang/gno v0.0.0-20240125181217-b6193518e278
-	github.com/gnolang/tx-archive v0.1.1
+	github.com/gnolang/gno v0.1.0-nightly.20240627
```

Comment:
btw, let's add a replace ../, nope?

Reply:
I am actually not sure of the dependency graph for this go module -- let me merge it as is, and open up another PR later if we can add the `replace` 🙏 

I'm super weary since this imports gno, but also other packages that import gno on their own (we had the same situation for `gnofaucet`, and needed to make it a separate `go.mod`)